lib/Interpreter/CIFactory.cpp

```diff
@@ -828,7 +828,11 @@ static void stringifyPreprocSetting(PreprocessorOptions& PPOpts,
 
     const size_t argc = COpts.Remaining.size();
     const char* const* argv = &COpts.Remaining[0];
-    std::vector<const char*> argvCompile(argv, argv+1);
+    std::vector<const char*> argvCompile;
+    if(argc && argv) {
+        argvCompile = std::vector<const char*>(argv, argv+1);
+    }
```

Comment:
It might be better/needed to have a non-empty default for argvCompile?

Reply:
I think this repo is just a mirror, consider moving your PR here instead: https://github.com/root-project/root/pulls (subdirectory interpreter/cling). Thanks!